### ISSUE TITLE
JAWS menu navigation keyboard

### ISSUE THREAD

**Issue by stef-barburou** [snap-framework/snap#13]
Page / URL: http://skynet-01.csps-efpc.com/_projects/Learnomatic/AQA/index_en.html?state=m0

Language: English

Tool(s): JAWS

Browser/Screen reader: IE

WCAG 2.0 reference: 2.1.1 keyboard
2.4.6 headings and labels

Details: AQA found that while using JAWS it reads out the following instruction for the menu item Resources.
"Toolbar has popup link toolbox press down to open toolbox"
One issue is that the down arrow on the keyboard does not actually open the toolbox. Which means the instructions are incorrect (2.4.6).

Another issue is that pressing the down key JAWS will read "menu". We further found that we cannot open the toolbox with any keys "space, arrow down, enter".
